lua/nvim-web-devicons.lua

```diff
@@ -1564,6 +1589,8 @@ local filetypes = {
   ["lock"] = "lock",
   ["lprolog"] = "sig",
   ["lua"] = "lua",
+  ["make"] = "GNUmakefile",
+  ["make"] = "Makefile",
```

Comment:
This one's a map of filetypes to icons... these two will be clobbered.

make -> makefile is all we need.